src/common/component/Input/Input.style.ts

```diff
@@ -0,0 +1,57 @@
+import { css } from '@emotion/react';
+
+import { InputProps } from '@/common/component/Input/Input';
+import { theme } from '@/common/style/theme/theme';
+
+export const inputContainerStyle = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '1.2rem',
+});
+
+export const inputWarpperStyle = css({
+  display: 'flex',
+  alignItems: 'center',
+
+  paddingLeft: '0.8rem',
+});
+
+export const inputStyle = css({
+  width: '100%',
+  paddingLeft: '1.2rem',
+  border: '0px',
+  backgroundColor: 'transparent',
+  weight: 400,
+  outline: 'none',
```

Comment:
css를 같은 그룹마다 한줄 띄고 쓰면 좋다는 의견들이 있었습니다! 한줄씩 띄어서 쓰면 styleLint 적용한것 같고 가독성도 좋드라구요~

Reply:
좋은 지적~ 좋은 말씀~ 입니당

Reply:
아 맞습니다 저도 그룹마다 띄워 봤는데 이게 다다른 그룹이라 모든 줄이 다 한줄씩 떨어진 상황이 되어서 오히려 이상해보여서 다시 없앴거든요.. 그래도 아래처럼 띄우는게 나을까요? 여러분들의 의견이 필요합니당..!!
```suggestion
  export const inputStyle = css({
  width: '100%',

  paddingLeft: '1.2rem',

  border: '0px',

  backgroundColor: 'transparent',

  weight: 400,
  
  outline: 'none',
});
```
``

Reply:
width / padding / outline 정도만 분리한다면 좋을 것 같네요! 보통 밑에서 함께 사용하는 `border`랑 `color`, `font` 관련 속성은 저 또한 구분 없이 쓰긴 합니다 !
그렇지만 다른 주요 속성들은 꼭 분리해주기!

+ 추가적으로 `weight`라는 스타일 속성은 뭔가요 ? `fontWeight`를 잘못쓰신 걸까요 ?

Reply:
그리고 왼쪽 패딩값을 준다면 오른쪽 패딩값을 줘서, 인풋에 텍스트를 꽉 차게 입력했을 시에도 좌우 여백이 동일하도록 설정하는게 좋을 것 같습니다 😄

Reply:
아.. fontWeight .. 였군요 하하 알려주셔서 감사합니다 😉 
글씨가 꽉 찼을 때 고려를 못했네요! 좋은 지적 감사합니닷! 👍